railties/lib/rails/commands/routes/routes_command.rb

```diff
@@ -8,6 +8,7 @@ class RoutesCommand < Base # :nodoc:
       class_option :controller, aliases: "-c", desc: "Filter by a specific controller, e.g. PostsController or Admin::PostsController."
       class_option :grep, aliases: "-g", desc: "Grep routes by a specific pattern."
       class_option :expanded, type: :boolean, aliases: "-E", desc: "Print routes expanded vertically with parts explained."
+      class_option :format, aliases: "-o", desc: "Print routes with specified format (currentry, json only)."
```

Comment:
I think to move "expanded" option to argument of  "--format" option is better but it's breaking changes...